src/main/app/src/interfaces.ts

```diff
@@ -0,0 +1,13 @@
+export interface ILandscape {
+  name?: string;
+  description?: string;
+  identifier?: string;
```

Comment:
Are there landscapes without ``name`` or ``identifier``?

Reply:
the landscape interface is something that we have to look on together, i didnt know what was really necessary and what not so i just made (almost) everything optional for now that could be made optional

Reply:
Landscapes must have identifiers and will very like have a name. The rest is indeed optional.